### PR TITLE
eoss recap blog

### PR DESCRIPTION
**Pull request by jhamman**
This PR adds our first blog, a brief recap of our CZI EOSS grant.

Pinging those mentioned by name for review: 

- @shoyer
- @rabernat
- @dcherian
- @benbovy
- @andersy005
- @alexamici
- @aurghs

_Note the rendered version of the site in the Vercel bot comment._

### CODE REVIEW

File: src/posts/czi-eoss-grant-conclusion/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Community Engagement
+
+More than any single feature addition, we felt like a key need for the Xarray project was a concerted focus on understanding and supporting our user community. To this end, we took on three tasks:
+
+1. [Anderson Banihirwe](https://github.com/andersy005) developed a new project website: [xarray.dev](https://xarray.dev) and [blog](https://xarray.dev/blog). This site includes a high-level introduction to Xarray, an interactive demo using [Pyodide](https://pyodide.org/en/stable/), and pointers to other cross-project resources. We also moved Xarray’s documentation site to a new URL: [docs.xarray.dev](https://docs.xarray.dev).
+2. We made numerous improvements to Xarray’s core documentation and extended [tutorial](https://tutorial.xarray.dev/intro.html). Notably, we reworked our [examples gallery](https://docs.xarray.dev/en/stable/gallery.html) and [tutorial page](https://docs.xarray.dev/en/stable/tutorials-and-videos.html).
```
Comment (jhamman) [inline src/posts/czi-eoss-grant-conclusion/index.md:16]: @andersy005 - please review and suggest any edits.

File: src/posts/czi-eoss-grant-conclusion/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+3. We completed Xarray’s first [user-survey](https://github.com/xarray-contrib/user-survey) in 2021. This gave us a detailed view of our user community and what their use-cases, priorities, and pain points are. Our [2022 user survey is live now](https://docs.google.com/forms/d/e/1FAIpQLSfnMd8UsC1XP1lPuFczl148VfpmwnFu4a0Z94odt1L6U0R0Pw/viewform?usp=sf_link), if you haven’t yet, please take a moment to fill it out. Keep an eye out for a future blog post unpacking the results from these surveys.
+
+## Flexible Grids and Indexes
+
+The second focus area in our proposal was a [long-awaited](https://docs.xarray.dev/en/v2022.03.0/roadmap.html#flexible-indexes) reworking of Xarray’s Indexes. [Benoît Bovy](https://github.com/benbovy) led this work which culminated in a very large [Pull Request](https://github.com/pydata/xarray/pull/5692) refactoring the internals of Xarray objects to explicitly include indexes are part of the data model. With the internal refactor in place, it is now possible to develop custom indexes for Xarray objects. Expect more on this new functionality in another blog post.
```
Comment (jhamman) [inline src/posts/czi-eoss-grant-conclusion/index.md:21]: @benbovy - please review and suggest any edits.

File: src/posts/czi-eoss-grant-conclusion/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Finally, the third focus area in our proposal was a refactor of Xarray’s storage backends to enable reading from and writing to a variety of storage formats. In this work, we wanted to standardize the backend API and support the development and integration of third-party backends through a plugin interface (i.e. [entrypoints](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#entry-points)). [Aureliana Barghini](https://github.com/aurghs) of [B-Open](https://github.com/aurghs) led this work which, starting with Xarray v0.18, added read-only support for custom backends. Since then, multiple third-party libraries have made use of this new functionality (e.g. [rioxarray](https://corteva.github.io/rioxarray/stable/getting_started/getting_started.html#rioxarray), [cfgrib](https://github.com/ecmwf/cfgrib)). For now, we’ve held off on adding entrypoint support for write operations but hope to see this progress in the [future](https://github.com/pydata/xarray/issues/5954). If you are interested in writing a custom backend, check out this [step-by-step guide](https://docs.xarray.dev/en/stable/internals/how-to-add-new-backend.html).
```
Comment (jhamman) [inline src/posts/czi-eoss-grant-conclusion/index.md:25]: @alexamici and @aurghs - please review and suggest any edits.

File: src/posts/czi-eoss-grant-conclusion/index.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+2. We made numerous improvements to Xarray’s core documentation and extended [tutorial](https://tutorial.xarray.dev/intro.html). Notably, we reworked our [examples gallery](https://docs.xarray.dev/en/stable/gallery.html) and [tutorial page](https://docs.xarray.dev/en/stable/tutorials-and-videos.html).
+3. We completed Xarray’s first [user-survey](https://github.com/xarray-contrib/user-survey) in 2021. This gave us a detailed view of our user community and what their use-cases, priorities, and pain points are. Our [2022 user survey is live now](https://docs.google.com/forms/d/e/1FAIpQLSfnMd8UsC1XP1lPuFczl148VfpmwnFu4a0Z94odt1L6U0R0Pw/viewform?usp=sf_link), if you haven’t yet, please take a moment to fill it out. Keep an eye out for a future blog post unpacking the results from these surveys.
+
+## Flexible Grids and Indexes
```
Comment (dcherian) [inline src/posts/czi-eoss-grant-conclusion/index.md:19]: Shall we add a one line description like "what  are indexes", "what are storage backends" as the second line?

**Review by dcherian — COMMENTED** [review]
Thanks @jhamman

File: src/posts/czi-eoss-grant-conclusion/index.md (hunk 1 above)
Comment (andersy005) [reply src/posts/czi-eoss-grant-conclusion/index.md:16]: apart from @dcherian's suggestion below, this section looks good.

**Review by rabernat — APPROVED** [review]

**Review by shoyer — APPROVED** [review]

**Comment by andersy005** [comment]
🥳 Hooray for the first blog post....  Thank you all! 🎉

File: src/posts/czi-eoss-grant-conclusion/index.md (hunk 2 above)
Comment (benbovy) [reply src/posts/czi-eoss-grant-conclusion/index.md]: Sorry @jhamman I've missed it but it looks great! Many thanks!

**Pull request merged**